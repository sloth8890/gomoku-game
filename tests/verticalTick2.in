place A1
place B1
place A5
place B2
place A2
place S1
place A4
place S2
place A3
