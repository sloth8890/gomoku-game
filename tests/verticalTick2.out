Black wins!
A1B1A5B2A2S1A4S2A3
Thank you for playing!
